radionuclide_scraper.py

```diff
@@ -73,167 +75,158 @@ def extract_table(bsobject):
                     break
                 elif table[i+5+k:i+5+k+5] == '</td>':
                     data.append(table[i+5:i+5+k].strip())
-                    break            
+                    break
         elif table[i:i+4] == '<td>':
             for k in range(100):
                 if table[i+4+k:i+4+k+1] == '*':
                     data.append('0')
                     break
                 elif table[i+4+k:i+4+k+3] == '<i>':
-                    data.append(table[i+4:i+4+k].strip())                
+                    data.append(table[i+4:i+4+k].strip())
                     for j in range(10):
                         if table[i+4+k+3+j:i+4+k+3+j+1] == '*':
                             data[-1] = '0'
                             break
                     break
                 elif table[i+4+k:i+4+k+5] == '</td>':
                     data.append(table[i+4:i+4+k].strip())
-                    break    
-            
+                    break
+
     return(data,table)
-    
-    
-    
+
+
 def extract_url(data):
-    
     """
     Extracting all intensities and url addons from the table.
-    """    
-      
+    """
     intensities = []
-    for i in range(1,len(data),5):
+    for i in range(1, len(data), 5):
         intensities.append(data[i])
-  
-    url_add_ons = []  
-    for i in range(4,len(data),5):
+
+    url_add_ons = []
+    for i in range(4, len(data), 5):
         for j in range(100):
             if data[i][j:j+4] == 'href':
                 for k in range(100):
                     if data[i][j+4+k:j+4+k+2] == '><':
                         url_add_ons.append(data[i][j+4+2:j+4+k-1])
-                        break        
+                        break
                 break
-           
     """
-    Removing all intensities that are less than 1% and their corresponding 
+    Removing all intensities that are less than 1% and their corresponding
     radionuclide url addon.
     """
-    
     for i in range(len(intensities)):
         if intensities[i] == '':
             intensities[i] = '0'
-        intensities[i] = float(intensities[i])   
+        intensities[i] = float(intensities[i])
         if intensities[i] < 1:
             intensities[i] = 0
-    #taking the index of each intensity that is less than 1%.         
+    """
+    taking the index of each intensity that is less than 1%.
+    """
```

Comment:
These statements are comments on the code, not docstrings. So you had it right before, they should be formatted using the `#` as a comment. The `"""` format is only for docstrings which are immediately under the function definition.